TFT/src/User/Configuration.h

```diff
@@ -148,6 +148,13 @@
  */
 #define TERMINAL_COLOR_SCHEME 0  // Default: 0
 
+/**
+ * Progress bar color
+ * The color of the progress bar during print
+ *   Options: [Orange: 0, Yellow: 1, Red: 2, Green: 3, Blue: 4, Cyan: 5, Magenta: 6, Purple: 7, Lime: 8, Gray: 9]
+ */
+#define PROGRESS_BAR_COLOR 0  // Default: 0
+
```

Comment:
you should move that setting after TERMINAL_KEYBOARD_LAYOUT (so in the param configurable at compile time).
Also please restore //#define MARKED_PROGRESS_BAR  // Default: commented (disabled). IOn this case // is not used for a comment but to disable a macro (or in general an instruction)

Reply:
Done.